.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 * @o3de/sig-docs-community-maintainers
 
 # Ownership owners
-/.github/CODEOWNERS @o3de/sig-chairs
+/.github/CODEOWNERS @o3de/sig-docs-community-maintainers
```

Comment:
I agree that this docs repo CODEOWNERS does not need to notify SIG Chairs on changes in ownership.